### ISSUE TITLE
 Illegal instruction (core dumped)

### ISSUE THREAD

**Issue by splurgein** [OpenNMT/OpenNMT-py#1327]
I am trying to run the default example after preprocessing when I try to train the model I get the below error 

python3 train.py -data data/demo -save_model demo-model

[2019-02-27 15:09:39,816 INFO]  * src vocab size = 24997
[2019-02-27 15:09:39,816 INFO]  * tgt vocab size = 35820
[2019-02-27 15:09:39,816 INFO] Building model...
[2019-02-27 15:09:43,349 INFO] NMTModel(
  (encoder): RNNEncoder(
    (embeddings): Embeddings(
      (make_embedding): Sequential(
        (emb_luts): Elementwise(
          (0): Embedding(24997, 500, padding_idx=1)
        )
      )
    )
    (rnn): LSTM(500, 500, num_layers=2, dropout=0.3)
  )
  (decoder): InputFeedRNNDecoder(
    (embeddings): Embeddings(
      (make_embedding): Sequential(
        (emb_luts): Elementwise(
          (0): Embedding(35820, 500, padding_idx=1)
        )
      )
    )
    (dropout): Dropout(p=0.3)
    (rnn): StackedLSTM(
      (dropout): Dropout(p=0.3)
      (layers): ModuleList(
        (0): LSTMCell(1000, 500)
        (1): LSTMCell(500, 500)
      )
    )
    (attn): GlobalAttention(
      (linear_in): Linear(in_features=500, out_features=500, bias=False)
      (linear_out): Linear(in_features=1000, out_features=500, bias=False)
    )
  )
  (generator): Sequential(
    (0): Linear(in_features=500, out_features=35820, bias=True)
    (1): Cast()
    (2): LogSoftmax()
  )
)
[2019-02-27 15:09:43,351 INFO] encoder: 16506500
[2019-02-27 15:09:43,351 INFO] decoder: 41613820
[2019-02-27 15:09:43,351 INFO] * number of parameters: 58120320
[2019-02-27 15:09:43,353 INFO] Starting training on CPU, could be very slow
[2019-02-27 15:09:43,353 INFO] Start training loop and validate every 10000 steps...
[2019-02-27 15:09:43,541 INFO] Loading dataset from data/demo.train.0.pt, number of examples: 10000
**Illegal instruction (core dumped)**

**Comment by vince62s** [comment]
it looks like an old pytorch problem.
what pytorch version are you using ?

**Comment by splurgein** [comment]
my pytorch version is 1.0.0a0+8322165

**Comment by vince62s** [comment]
try to remove your pytorch install and reinstall making sure you take the accurate wheel corresponding to your cuda install.

**Comment by vince62s** [comment]
closing, reopen if needed.
